docs/src/Tooltip.doc.js

```diff
@@ -1,165 +1,822 @@
+        cardSize="md"
+        type="do"
+        description="Use Tooltip to add supplementary information about a feature, typically paired with an `info-circle` [IconButton](/IconButton)."
+        defaultCode={`
+<Tooltip inline text="Total ad spend in the select time period">
+  <Icon
+    accessibilityLabel="Additional info."
+    icon="info-circle"
+    inline
+    color="gray"
+    size={16}
+  />
+  </Tooltip>
+`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description="Pair a Tooltip with a disabled element. See [disabled items](#Disabled-itemds) to learn more."
+        defaultCode={`
 <Tooltip
-  inline
-  text="Logout"
   link={
-    <Link href="https://pinterest.com" target="blank">
-      <Text color="white" size="sm" weight="bold">
+    <Text color="white" size="sm" weight="bold">
+      <Link
+        href="https://help.pinterest.com/en/business/article/get-a-business-account"
+        target="blank"
+      >
         Learn more
-      </Text>
-    </Link>
+      </Link>
+    </Text>
   }
+  text="There was a problem converting to a personal account."
 >
-  <Icon
-    accessibilityLabel="Logout of your profile"
-    color="pine"
-    icon="logout"
+  <Button size="lg" disabled text="Convert to personal account" />
+</Tooltip>
+`}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection
+    name="Accessibility"
+    description={`
+When using Tooltip with [IconButton](/IconButton), avoid repetitive labeling. The accessibilityLabel provided to IconButton should describe the intent of the button, not the icon itself. For instance, use “Settings” instead of “Cog icon”. Tooltip text can expand upon that intention, as seen with the cog IconButton. If the Tooltip text and IconButton accessibilityLabel contain the same content, pass an empty string to \`accessibilityLabel\`, as seen with the send IconButton.`}
+  >
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="md"
+        defaultCode={`
+<Tooltip text="Customize performance stats for your paid ads">
+  <IconButton
+    accessibilityLabel="Settings"
+    bgColor="white"
+    icon="cog"
+    iconColor="darkGray"
+    size="lg"
+  />
+</Tooltip>
+`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        defaultCode={`
+<Tooltip text="Send Pin">
+  <IconButton
+    accessibilityLabel=""
+    bgColor="white"
+    icon="share"
+    iconColor="darkGray"
+    size="lg"
   />
 </Tooltip>
+`}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection
+    name="Disabled elements"
+    description={`
+Tooltips must be paired with an interactive, focusable element, like [Button](/Button) or [IconButton](/IconButton). They cannot be paired with anything disabled, because this prevents keyboard users from triggering Tooltip and consuming its content. To test if you’re using a Tooltip properly, use your keyboard rather than your mouse to trigger Tooltip.
```

Comment:
One tiny fix: "They cannot be paired with anything disabled, because"... becomes "They cannot be paired with anything disabled or static, because"

Reply:
DONE